### ISSUE TITLE
Can't search github

### ISSUE THREAD

**Issue by Naatan** [StractOrg/stract#141]
Stract is supposedly for tinkerers and hackers, which aligns well with what github is used for. But I can't find a way to have it actually offer me search results for Github project. Starting with this project itself, try searching for `stract githbub` or `stract site:github.com` and you won't find what you're looking for.

I understand this search engine is relatively new, and perhaps it just hasn't gotten to that level of indexing yet.

**Comment by bendangelo** [comment]
I don't think this is possible because Github search requires logging in now. So how can it be indexed?

**Comment by Naatan** [comment]
I'm not suggesting stract do an actual github search, I'm suggesting stract indexes github so it can be used to search github.

**Comment by YeonwooSung** [comment]
Just curious, any progress here?

**Comment by mikkeldenker** [comment]
I don't think there are any technical issues specifically related to github as far as I know. There are ~9500 pages from github in the current index, so it's mostly an issue of having a larger index so all the expected repositories are included in the index
